codecov.yml

```diff
@@ -1,3 +1,4 @@
 ignore:
+  - "api/**/*.generated.go"
```

Comment:
The file `api/graphql/resolvers/*.go` (except `api/graphqlresolvers/*.util.go`) are not fully generated go files. `gqlgen` generates functions without implementation, see [the example](https://github.com/photoview/photoview/blob/master/api/graphql/resolvers/album.go#L67) (This is also not implemented in [the original file](https://github.com/photoview/photoview/blob/c151bdee963c97c6f0c7561eda32974845625bb5/api/graphql/resolvers/album.go#L124)). We added implementation. `gqlgen` will move the implementation to the new file when regenerating them.

I still think we should cover those files, especially they are manipulating data. I'd like to move those db methods to a database layer, but it's not an easy work.